goruntime.go

```diff
@@ -86,27 +86,27 @@ func goToolsAtErr(path PathString) (*GoTools, error) {
 }
 
 func initGoInst() (*GoTools, error) {
+	return initGoInstNamed("go")
+}
+
+func initGoInstNamed(goName string) (*GoTools, error) {
 	_goMux.Lock()
 	defer _goMux.Unlock()
 	if _goInstance != nil {
 		return _goInstance, nil
 	}
 	goRootPath := cache.New(func() (string, error) {
-		goRootDir, ok := os.LookupEnv("GOROOT")
```

Comment:
No longer resolving `GOROOT` from the environment. This affordance is a bit more trouble than it's worth.